scrapers_next/vt/committees.py

```diff
@@ -0,0 +1,73 @@
+from lxml.html import fromstring
+from lxml.html.clean import Cleaner
+from lxml.etree import ParserError
+from spatula import JsonListPage, URL, SkipItem
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeList(JsonListPage):
+    source = URL(
+        "https://legislature.vermont.gov/committee/loadList/2024/",
+        timeout=10,
+    )
+
+    def process_page(self):
+        for each_committee in self.response.json()["data"]:
+            # name
+            name = each_committee["LongName"]
+
+            # chamber
+            if "house" in name.lower():
+                chamber = "lower"
+            elif "senate" in name.lower():
+                chamber = "upper"
+            else:
+                chamber = "legislature"
+
+            # classification & parent check
+            classification = "committee"
+            parent = None
+            if "subcommittee" in name.lower():
+                classification = "subcommittee"
+                parent = name.lower().split()[-1].replace(")", "")
+
+            # extracting members
+            members_html = each_committee["Members"].strip()
+            members_html = members_html.replace("</br>", "\n")
+            try:
+                doc = fromstring(members_html)
+            except ParserError:
+                SkipItem(f"No membership data found for: {name}")
```

Comment:
Given that you're running this scraper in a `process_page()` method rather than `process_item()`, the `SkipItem()` class isn't appropriate here.
- Change it to `logging.warning(f"No membership data found for: {name}")` instead (you'll need to `import logging` and the top and also remove `SkipItem` from your imports)